js/sdk/src/components/ReleaseSettingsModal.js

```diff
@@ -66,7 +66,11 @@ const ReleaseSettingsModal = ({
             {userIsRecipient && (
               <Royalty releasePubkey={releasePubkey} release={release} />
             )}
-
+            <ShareToTwitter
+              artist={metadata.properties.artist}
+              title={metadata.properties.title}
+              url={metadata.external_url?.slice(8)}
```

Comment:
why are we using the `.slice(8)` on the url?
if its required for the link to behave properly, it should be done in the component for reusability

Reply:
yeah this is confusing - if you are trying to remove `https://` you should probably use replace so it is more explicit as to what the purpose is.... i also just dont think you need to remove that at all and that is fine in the tweet. all tweets turn into tiny links anyway so all links end up taking up the same amount of characters